### DISCUSSION TITLE
推薦的核心通識

### DISCUSSION THREAD

**Discussion by yubo0oo** [1101-GS4719-C/Python-Community-GS4719-C#2023] [General]
我的通識修滿了
但是就差一個核心通識
有沒有推薦的超凉核心通識

**Comment by yue0903ZZ** [comment]
天文學導論，冷氣很涼課也很涼，前兩節不用點名，第三節也只是去聽別人報告而已，很推。

**Comment by yue0903ZZ** [comment]
天文學導論聽起來超讚！冷氣涼爽又輕鬆，還不用點名，聽別人報告也能學到不少，整堂課氛圍輕鬆又有趣，真的很適合放鬆又吸收知識，推薦！

**Comment by DDuopg** [comment]
如果是在意成績的人我推薦可以修化學與生活，上課回答老師問題或是建議老師期中期末考題都可以得到額外加分，我上學期修拿到了107分。而且上課真的可以認識到蠻多生活中的化學知識的，像我上完就多了很多東西不敢吃

**Comment by DDuopg** [comment]
然後我之前有修過全球環境變遷跟法律的世界，都被我期中停修掉了。全球環境變遷是因為我真的不知道他們在上什麼，感覺很像在安麗自己的研究室而已，法律的世界是因為我聽說給分很甜，結果考完期中只有6個人85以上(全班120人)，最高的只拿92，感覺是助教出的題目，都open book了完全找不到答案在哪，有種被法律人排擠在法律門外的感覺
